src/data/faq_de.json

```diff
@@ -180,12 +180,15 @@
                         ]
                     },
                     {
-                        "title": "Kann ein vollständiger Impfstatus in der App als Berechtigung zum Reisen dienen?",
+                        "title": "Kann mein Zertifikat in der App als Berechtigung zum Reisen dienen?",
                         "anchor": "vac_cert_privileges_travel",
                         "active": true,
                         "textblock": [
-                            "Ob Sie mit einem vollständigen Impfschutz beispielsweise reisen dürfen, hängt von den Bestimmungen des jeweiligen Landes ab.",
-                            "Sie können unter anderem dieses <a href='https://reopen.europa.eu/de' target='_blank' rel='noopener noreferrer'>Webseite</a> benutzen, um ihre Reise innerhalb der EU zu planen."
+                            "Der digitale Impfnachweis kann nicht nur in Deutschland genutzt werden. Der in Deutschland ausgestellte digitale Impfnachweis kann auch im EU-Ausland als Nachweis für erfolgte Impfungen gelten, wenn der jeweilige EU-Mitgliedstaat digitale Nachweise zulässt. Dasselbe gilt perspektivisch auch für digitale Genesenenzertifikate und Testzertifikate. Im Allgemeinen prüfen die jeweiligen EU-Mitgliedstaaten mit ihren eigenen Prüf-Apps Ihr Zertifikat, d. h. es ist unerheblich, in welcher Sprache Ihr Zertifikat auf Ihrem Gerät dargestellt wird.",

```

Comment:
```suggestion
                            "Der digitale Impfnachweis kann nicht nur in Deutschland genutzt werden. Der in Deutschland ausgestellte digitale Impfnachweis kann auch im EU-Ausland als Nachweis für erfolgte Impfungen gelten, wenn der jeweilige EU-Mitgliedstaat digitale Nachweise zulässt. Dasselbe gilt perspektivisch auch für digitale Genesenenzertifikate und Testzertifikate. Im Allgemeinen prüfen die jeweiligen EU-Mitgliedstaaten mit ihren eigenen Prüf-Apps Ihr Zertifikat, d.&nbsp;h. es ist unerheblich, in welcher Sprache Ihr Zertifikat auf Ihrem Gerät dargestellt wird.",
```

Reply:
thanks. I will close this issue, which became stale. A new PR has been submitted. Sorry for the inconvenience .